### PR TITLE
add new gin-gonic source gopkg.in/gin-gonic/gin.v1

### PR DESCRIPTION
**Pull request by marinsalinas**

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -1,11 +1,10 @@
 package location
 
 import (
+	"gopkg.in/gin-gonic/gin.v1"
```
Comment (appleboy) [inline config.go]: Please move to footer.

File: example/custom/example.go [hunk 2]
```diff
@@ -1,8 +1,8 @@
 package main
 
 import (
-	"github.com/gin-contrib/location"
-	"github.com/gin-gonic/gin"
+	"github.com/marinsalinas/location"
```
Comment (appleboy) [inline example/custom/example.go]: Wrong path?

File: example/default/example.go [hunk 3]
```diff
@@ -1,8 +1,8 @@
 package main
 
 import (
-	"github.com/gin-contrib/location"
-	"github.com/gin-gonic/gin"
+	"github.com/marinsalinas/location"
```
Comment (appleboy) [inline example/default/example.go]: Same as above.

File: config.go (hunk 1 above)
Comment (tboerger) [inline config.go]: Yeah, first stdlib and than other sources

**Pull request closed without merging**